src/QirRuntime/test/QIR-static/qir-test-strings.qs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Testing.QIR.Str {
+
+    open Microsoft.Quantum.Intrinsic;
+
+    function PauliToStringTest() : Int {
+
+        mutable pauli   = PauliI;
+        mutable str     = $"Pauli value: {pauli}";
+        if "Pauli value: PauliI" != str      { return 1; }     // The return value indicates which test case has failed.
```

Comment:
You may be able to avoid mutability here, since `pauli` and `str` are each only used once:

```suggestion
        if "Pauli: PauliI" != $"Pauli: {PauliI}" { return 1; }     // The return value indicates which test case has failed.
```

Reply:
Fixed.